bin/generate_npm_tarball.sh

```diff
@@ -6,13 +6,27 @@ if [ $# -ne 2 ]; then
 fi
 package=$1
 output=$2
-
 wd=$(mktemp -d)
 trap "rm -r '$wd'" EXIT INT TERM
 
-mkdir $wd/cache $wd/install
+create_cache() {
+  node_modules/npm/bin/npm-cli.js install --cache $wd/cache $package --production --verbose
+}
+
+mkdir $wd/cache-combined $wd/cache $wd/install
 cd $wd/install
-npm install --cache $wd/cache $package --production --verbose
+npm install npm@1.x
```

Comment:
I think this has to be 1.3.x, as 1.4.x onwards seems to have the newer cache style